app/classifier/tasks/survey/translations.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import merge from 'lodash/merge';
+import translations from '../../../pages/project/translations';
+
+function SurveyTranslations(props) {
+  const { task } = props;
+  const taskStrings = translations.strings.workflow.tasks;
+  const translation = merge({}, task);
```

Comment:
@simoneduca updated to use deep merge to copy the task structure over to the translation.

The maps are still here to copy the translated strings across, if they exist, but there's probably a better way to do that too.

Reply:
Of course, I forgot about lodash. Should read comments better. It's made a nice difference 👍